verticaltabs.js

```diff
@@ -487,7 +487,10 @@ VerticalTabs.prototype = {
     let pin_button = this.createElement('toolbarbutton', {
       'id': 'pin-button',
       'tooltiptext': 'Keep sidebar open',
-      'onclick': `let box = document.getElementById('main-window');
+      'onclick': ` if (event.which == 3) {
```

Comment:
Extra space in front of the `if` here…  Also, I think we want to use `===`, don’t we?